assets/pages/regular-contributions/components/formFields.jsx

```diff
@@ -158,17 +160,20 @@ function NameForm(props: PropTypes) {
 
   return (
     <form className="regular-contrib__name-form">
-      <TextInput
-        id="email"
-        value={props.email.value}
-        labelText="Email"
-        placeholder="Email"
-        onChange={props.setEmail}
-        onBlur={onFormFieldBlur(props.setEmailShouldValidate, continueButtonClassName)}
-        modifierClasses={['email']}
-        showError={shouldShowError(props.email)}
-        errorMessage="Please enter a valid email address."
-      />
+      {!props.isSignedIn ? (
+        <TextInput
+          id="email"
+          value={props.email.value}
+          labelText="Email"
+          placeholder="Email"
+          onChange={props.setEmail}
+          onBlur={onFormFieldBlur(props.setEmailShouldValidate, continueButtonClassName)}
+          modifierClasses={['email']}
+          showError={shouldShowError(props.email)}
+          errorMessage="Please enter a valid email address."
+        />
+      ) : null
+      }
```

Comment:
can this be on the line above?

Reply:
or maybe not necessarily that solution - but the fact that `) : null` and `}` are at the same indentation level feels wrong to me